css/view.css

```diff
@@ -0,0 +1,4 @@
+.jobs-unified-top-card__subtitle-secondary-grouping
```

Comment:
Selects the child element with class `.jobs-unified-top-card__bullet` of a parent element with class `.jobs-unified-top-card__subtitle-secondary-grouping`.